packages/turf-area/index.ts

```diff
@@ -23,10 +23,13 @@ export default function area(
   return geomReduce(
     geojson,
     (value, geom) => {
+      if (geom == null) {
+        return value;
+      }
       return value + calculateArea(geom);
     },
     0
-  );
+  ) as number;
```

Comment:
Similar to my comment on turf-length, if this comes back as undefined (empty feature collection or null geometries) then consider the area to return should be zero

Reply:
If you pass a default value of zero it will definitely return a number.